components/tikv_util/src/time.rs

```diff
@@ -284,6 +284,12 @@ pub enum Instant {
     MonotonicCoarse(Timespec),
 }
 
+impl Default for Instant {
```

Comment:
Why not just call `Instant::now()`? Note it's also not implemented for std::time::Instant.